### DISCUSSION TITLE
PDF da ficha simplificada

### DISCUSSION THREAD

**Discussion by Bilhalva1** [YuriAlessandro/gerador-ficha-tormenta20#335] [Ideias]
seria incrível se a ficha simplificada fosse disponibilizada como pdf editável, facilitaria muito a criação de npcs e até a criação de fichas pra oneshots

**Comment by n-malll** [comment]
Se der pra baixar como PNG para usar pra NPCs fica mt bom, já que não precisa de muita coisa dependendo do que for fazer ocm ele

**Reply by YuriAlessandro** [reply]
Para baixar como PNG ou JPG não é só tirar um print?

**Reply by n-malll** [reply]
eu comentei errado era PDF rs, como se fosse uma ficha normal

**Comment by YuriAlessandro** [comment]
@Bilhalva1 eu adicionei a funcionalidade de imprimir ficha simplificada hoje.
